source/whats-new.txt

```diff
@@ -0,0 +1,54 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+* :ref:`Version 4.2 <version-4.2>`
+* :ref:`Version 4.1 <version-4.1>`
+* :ref:`Version 4.0 <version-4.0>`
+
+
+.. _version-4.2:
+
+What's New in 4.2
+-----------------
+New features of the 4.2 Java driver release include:
+
+- Enabled Azure and GCP key stores to client-side field level encryption
+- Enabled Kerberos caching tickets for reuse in multiple authentication requests
+- `com.mongodb.MongoClient <http://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/MongoClients.html>`_ instances with ``MongoClientSettings`` or ``ConnectionString`` as the configuration
+-  ``explain()`` can now be used on `find <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/FindIterable.html#explain()>`_ and `aggregate <https://mongodb.github.io/mongo-java-driver/4.2/apidocs/mongodb-driver-sync/com/mongodb/client/AggregateIterable.html#explain()>`_ commands
```

Comment:
Consider adding "The ``explain()`` method..." so that the list item doesn't start with a lower case word.